build/sbt

```diff
@@ -1,60 +1,75 @@
 #!/usr/bin/env bash
 
-#
-# Licensed to the Apache Software Foundation (ASF) under one or more
-# contributor license agreements.  See the NOTICE file distributed with
-# this work for additional information regarding copyright ownership.
-# The ASF licenses this file to You under the Apache License, Version 2.0
-# (the "License"); you may not use this file except in compliance with
-# the License.  You may obtain a copy of the License at
-#
-#    http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-
-# When creating new tests for Spark SQL Hive, the HADOOP_CLASSPATH must contain the hive jars so
-# that we can run Hive to generate the golden answer.  This is not required for normal development
-# or testing.
-for i in "$HIVE_HOME"/lib/*
-do HADOOP_CLASSPATH="$HADOOP_CLASSPATH:$i"
-done
-export HADOOP_CLASSPATH
+
+###  ------------------------------- ###
+###  Helper methods for BASH scripts ###
+###  ------------------------------- ###
 
 realpath () {
 (
   TARGET_FILE="$1"
+  FIX_CYGPATH="$2"
```

Comment:
Do you know if the automated testing cover CYGPATH? If not, we may need to get some community help to test this since I assume we haven't test against building with cygwin.

Reply:
mmh so I had upgraded sbt to a more recent version and that's why it's now supporting CYGPATH. I don't think we ever use it in our testing workflow. If you think it's harmful to leave it here I can revert sbt to its previous version but it seemed compatible in testing and packaging.

Reply:
I reverted sbt to its previous version as its behaviour was slightly different, let's upgrade in a different PR